ClinGen/clincoded
# Connects to #1314. Connects to #1401. Updates to adding PMIDs and displaying their dates.

PMIDs of articles published before the 10th of the month:
29088396
29088483
29207444

Criteria for a valid prefix (to a PMID being added):
* Starts with the abbreviation "pmid" (case-insensitive).
* Includes a colon between "pmid" and the ID itself.
* Includes zero or more whitespace before and after the colon.

Steps to test #1314 and #1401 simultaneously:
1. Select a gene-disease record in the GCI.
2. Click on the "Add New PMID" button.
3. Add one of the PMIDs provided above, with or without a valid prefix (like “PMID:#”, “PmId :#” or “pmid: #”.
4. Once the article has been retrieved or added, confirm that the day of the month doesn't start with a zero.
